Tests/SourceKittenFrameworkTests/Fixtures/Realm/Stub.m

```diff
@@ -0,0 +1 @@
+#import <Foundation/Foundation.h>
```

Comment:
Why is this file needed?

Reply:
It's part of the dummy xcodeproject I added to the fixtures to check the Realm headers.  If there are no .m files in the project then there is no objective-C compiling in the xcodebuild output.

Reply:
ok